openstack/octavia/templates/secrets.yaml

```diff
@@ -17,3 +17,7 @@ type: Opaque
 data:
   secrets.conf: |
     {{ include (print .Template.BasePath "/etc/_secrets.conf.tpl") . | b64enc | indent 4 }}
+{{- if .Values.osprofiler.enabled }}
+  osprofiler.conf: |
```

Comment:
I would prefer to do [the same](https://github.com/search?q=repo%3Asapcc%2Fhelm-charts+%22%7B%7B-+if+.Values.osprofiler.enabled+%7D%7D%22&type=code) and keep this configuration as part of `_secrets.conf.tpl` file instead of create a new one.